include/ensmallen_bits/ada_sqrt/ada_sqrt.hpp

```diff
@@ -0,0 +1,168 @@
+/**
+ * @file ada_sqrt.hpp
+ * @author Marcus Edel
+ *
+ * Implementation of the AdaSqrt optimizer. AdaSqrt is an optimizer that
+ * chooses learning rate dynamically by adapting to the data and iteration.
+ *
+ * ensmallen is free software; you may redistribute it and/or modify it under
+ * terms of the 3-clause BSD license. You should have received a copy of the
+ * 3-clause BSD license along with ensmallen. If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef ENSMALLEN_ADA_SQRT_ADA_SQRT_HPP
+#define ENSMALLEN_ADA_SQRT_ADA_SQRT_HPP
+
+#include <ensmallen_bits/sgd/sgd.hpp>
+#include "ada_sqrt_update.hpp"
```

Comment:
The style of these two `#inlcude` statements differs, which smells weird.
Note that both gcc and clang (and probably other compilers) interpret `#include <file>` and `#include "file"` slightly differently.  The former (with angle brackets) means that usual system paths (and any paths specified via `-I`) are to be searched first, possibly omitting the current directory.  The latter (with double quotes) means that the current directory (where the present file being processed resides) is to be searched first.  

So having two separate styles for `#include` here looks like a bug.  At best this may blow up with `file not found` during compilation.  At worst this may lead to subtle errors if a user happens to have two separate installations of ensmallen on their system (say an old one in `/usr/include`).

Reply:
Good point, I don't mind to change it, especially since I mainly used it for convenience, that said we used the different styles in some other places as well, so I'll open another PR to unify the style across the code base.